src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -305,10 +305,8 @@ def _update_launch_string_wrt_gui_options(
         show_gui is None and (os.getenv("PYFLUENT_SHOW_SERVER_GUI") != "1")
     ):
         if (
```

Comment:
if not {"-g", "-gu"} & set(additional_arguments.split()):

Reply:
@seanpearsonuk , Thank you for this. I was really trying to make this a one-liner :)